### PR TITLE
Add TextField component to the sistent components page #5948

### PR DESCRIPTION
**Pull request by nooras**
**Description**
Add TextField component to the sistent components page

This PR fixes #
#5948 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 
Screenshot:
screencapture-localhost-8000-projects-sistent-components-2024-10-19-20_19_31

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bfd7e4ca470fae778d1c2a8aaa75b0d52361625e at: https://6713d1eb18a026d785930e4c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 76aae3371601356825f07b8c693d1fe1c30d30ae at: https://6713d883ee5caad9b39ed31e--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@nooras 
Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by vishalvivekm** [comment]
> @nooras Thank you for your contribution! Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).
> 
> Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

adding

File: src/sections/Projects/Sistent/components/text-field/index.js [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+
+const SistentTextField = () => {
+  const location = useLocation();
+  const { isDark } = useStyledDarkMode();
+
+  return (
+    <SistentLayout title="TextField" >
+      <div className="content">
+        <a id="Identity">
+          <h2>TextField</h2>
```
Comment (sudhanshutech) [inline src/sections/Projects/Sistent/components/text-field/index.js:18]: Text Field

File: src/sections/Projects/Sistent/components/text-field/code-block.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+import Code from "../../../../../components/CodeBlock";
+export const CodeBlock = ({ name, code }) => {
```
Comment (sudhanshutech) [inline src/sections/Projects/Sistent/components/text-field/code-block.js:3]: @nooras isn't code block component already there then why creating new?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Review by sudhanshutech — CHANGES_REQUESTED** [review]
add some space
image

**Comment by sudhanshutech** [comment]
merge conflicts...

**Comment by l5io** [comment]
🚀 Preview for commit d2c1f526724894b413b00ed2842f7e42a7df71aa at: https://671939227bdcdc52dccce67e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ff800ee5f638fa371fbdddcf5f7d363a0bae9661 at: https://6719f7a3f1138a8875d68370--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**